src/buildHTML.js

```diff
@@ -1095,33 +1101,30 @@ groupTypes.sqrt = function(group, options) {
     const minDelimiterHeight = (inner.height + inner.depth +
         lineClearance + theta) * options.sizeMultiplier;
 
-    // Create a \surd delimiter of the required minimum size
-    const delimChar = delimiter.customSizedDelim("\\surd", minDelimiterHeight,
-            false, options, group.mode);
-    const delim = makeSpan(["sqrt-sign"], [delimChar], options);
+    // Create a sqrt SVG of the required minimum size
+    const img = delimiter.customSizedDelim("\\surd", minDelimiterHeight,
+                    false, options, group.mode);
+
+    const surdWidth = img.width + "em";
+    // Actually img width will be set by CSS to match argument width.
```

Comment:
nit: this comment should go above line 1108 to match the position of the other comments and remove "Actually"